src/unlimited.cpp

```diff
@@ -660,6 +663,16 @@ void UnlimitedSetup(void)
 
 }
 
+void UnlimitedCleanup()
+{
+    CStatBase* obj = NULL;
+    while (!mallocedStats.empty())
+    {
+        obj = mallocedStats.front();
+        delete obj;
+        mallocedStats.pop_front();
+    }
+}
```

Comment:
Prefer `= nullptr;`. Also see my comment above on `unique_ptr` which could make `UnlimitedCleanup` unnecessary unless the intent is to log this for statical purposes?